tests/setup.js

```
@@ -0,0 +1,99 @@
+		removeItem(k) { delete store[k]; },
+		clear() { store = {}; },
+		key(n) { return Object.keys(store)[n] ?? null; },
+		get length() { return Object.keys(store).length; }
+	};
+}
+const memStorage = createMemoryStorage();
+
+// Polyfill global localStorage for jsdom consistency - ensure bare `localStorage` works in hooks.client.js
+if (typeof window !== 'undefined') {
+	if (!window.localStorage) window.localStorage = memStorage;
+	if (typeof globalThis.localStorage === 'undefined') globalThis.localStorage = window.localStorage;
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)